components/org.wso2.carbon.identity.organization.management.service/src/test/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImplTest.java

```diff
@@ -722,6 +723,26 @@ public void testGetChildOrganizationIds() throws Exception {
         Assert.assertTrue(recursiveChildIds.contains(ORG3_ID));
     }
 
+    @Test
+    public void testGetOrganizationIdToNameMap() throws OrganizationManagementException {
```

Comment:
add negative cases as well.
when the org list is empty, when the org list contains invalid org id